embassy-nrf/src/radio/mod.rs

```diff
@@ -6,12 +6,42 @@
 #![macro_use]
 
 /// Bluetooth Low Energy Radio driver.
+#[cfg(not(feature = "nrf51"))]
 pub mod ble;
+#[cfg(any(
+    feature = "nrf52820",
+    feature = "nrf52833",
+    feature = "nrf52840",
+    feature = "_nrf5340-net"
```

Comment:
The nrf52811 also supports the IEE 802.15.4 according to [the datasheet](https://infocenter.nordicsemi.com/pdf/nRF52811_PS_v1.0.pdf) (6.14.13)